spring-core/src/main/java/org/springframework/util/comparator/ComparableComparator.java

```diff
@@ -35,8 +35,13 @@
 	 * @see Comparators#comparable()
 	 */
 	@SuppressWarnings("rawtypes")
-	public static final ComparableComparator INSTANCE = new ComparableComparator();
-
+	private static final ComparableComparator INSTANCE = new ComparableComparator();
+	
+	private ComparableComparator(){}
+	
+	public ComparableComparator getInstance(){
```

Comment:
have a look [https://github.com/spring-projects/spring-framework/pull/25478/files](here). This class should be deprecated and not longer be updated, at all.

Reply:
No, as this class is present in the repository, there is a valid reason to update this as per the latest design. If you feel , you can raise different PR if required.